apps/studio/pages/project/[ref]/auth/smtp.tsx

```diff
@@ -0,0 +1,75 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+
+import { SmtpForm } from 'components/interfaces/Auth'
+import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import { useParams } from 'common'
+import { useCurrentPath } from 'hooks/misc/useCurrentPath'
+import Link from 'next/link'
+import { NavMenu, NavMenuItem } from 'ui'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldDivider,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
+import type { NextPageWithLayout } from 'types'
+
+const PageLayout: NextPageWithLayout = () => {
+  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+  const isPermissionsLoaded = usePermissionsLoaded()
+  const { ref } = useParams()
+  const currentPath = useCurrentPath()
+
+  const navMenuItems = [
+    {
+      label: 'Templates',
+      href: `/project/${ref}/auth/templates`,
+    },
+    {
+      label: 'SMTP Settings',
+      href: `/project/${ref}/auth/smtp`,
+    },
+  ]
+
+  if (isPermissionsLoaded && !canReadAuthSettings) {
+    return <NoPermission isFullPage resourceText="access your project's email settings" />
+  }
+
+  return (
+    <div>
+      <ScaffoldHeader className="pb-0">
+        <ScaffoldContainer id="auth-page-top">
+          <ScaffoldTitle>Email Settings</ScaffoldTitle>
+          <ScaffoldDescription>
+            Configure what emails your users receive and how they are sent
+          </ScaffoldDescription>
+          <NavMenu
+            className="border-none max-w-full overflow-y-hidden overflow-x-auto mt-4"
+            aria-label="Auth email settings navigation"
+          >
+            {navMenuItems.map((item) => (
+              <NavMenuItem key={item.label} active={currentPath === item.href}>
+                <Link href={item.href}>{item.label}</Link>
+              </NavMenuItem>
+            ))}
+          </NavMenu>
+        </ScaffoldContainer>
+      </ScaffoldHeader>
+
+      <ScaffoldDivider />
+
+      <ScaffoldContainer className="my-8 space-y-8">
+        <SmtpForm />
```

Comment:
This callout is missing on the new page
![image](https://github.com/user-attachments/assets/f74017a5-4dfd-4060-ae8a-768ac387e13e)

Reply:
I removed this intentionally but can add back if we think it helps. It's basically redirecting to itself and the warning is on templates page which is now in tab next to smtp.

Reply:
ahhh okay gotcha yeah that makes sense! okay looks good then :) minor one - the button could probably go on a next line

i can look into that for you

Reply:
okay done! resolving